docs/api-guide.md

```diff
@@ -124,43 +137,45 @@ admin.createTable("ns", "tbl", tableMetadata, options);
 You can create a secondary index as follows:
 
 ```java
-// Create a secondary index on a column "c5" of a table "ns.tbl". It will throw an exception if the secondary index already exists
+// Create a secondary index on column "c5" for table "ns.tbl". If a secondary index already exists, an exception will be thrown.
 admin.createIndex("ns", "tbl", "c5");
 
-// Create a secondary index only if it does not already exist
+// Create the secondary index only if it does not already exist.
 boolean ifNotExists = true;
 admin.createIndex("ns", "tbl", "c5", ifNotExists);
 
-// Create a secondary index with options
+// Create the secondary index with options.
 Map<String, String> options = ...;
 admin.createIndex("ns", "tbl", "c5", options);
 ```
 
 ### Add a new column to a table
 
-You can add a new non-partition key column to a table as follows:  
+You can add a new, non-partition key column to a table as follows:
+
 ```java
-// Add the new column "c6" of type INT to the table "ns.tbl"  
+// Add a new column "c6" with the INT data type to the table "ns.tbl".
 admin.addNewColumnToTable("ns", "tbl", "c6", DataType.INT)
 ```
 
-This should be executed with significant consideration as the execution time may vary greatly
-depending on the underlying storage. Please plan accordingly especially if the database runs in production:
-- For Cosmos DB for NoSQL and DynamoDB: this operation is almost instantaneous as the table
-     schema is not modified. Only the table metadata stored in a separated table are updated.
-- For Cassandra: adding a column will only update the schema metadata and do not modify existing 
-  schema records. The cluster topology is the main factor for the execution time. Since the schema 
-  metadata change propagates to each cluster node via a gossip protocol, the larger the cluster, the
-  longer it will take for all nodes to be updated.
-- For relational databases (MySQL, Oracle, etc.): it may take a very long time to execute and a 
-      table-lock may be performed.
+{% capture notice--info %}
+**Note**
+
+You should carefully consider adding a new column to a table because the execution time may vary greatly depending on the underlying storage. Please plan accordingly and consider the following, especially if the database runs in production:
+
+- **For Cosmos DB for NoSQL and DynamoDB:** Adding a column is almost instantaneous as the table schema is not modified. Only the table metadata stored in a separate table is updated.
+- **For Cassandra:** Adding a column will only update the schema metadata and will not modify the existing schema records. The cluster topology is the main factor for the execution time. Changes to the schema metadata are shared to each cluster node via a gossip protocol. Because of this, the larger the cluster, the longer it will take for all nodes to be updated.
+- **For relational databases (MySQL, Oracle, etc.):** Adding a column may take a very long time to execute. In addition, a table lock may occur.
```

Comment:
This is not a direct comment on this PR, but is this the case if a newly added column is set to NULLable and defaults to NULL? I suppose underlying RDBMSs normally take not so long to add a NULL column.

Reply:
I don't think adding a NULL column takes a long time.

@Torch3333 Do you have any comment ton this?

Reply:
Not really; I don't know the answer to that.

Reply:
Thanks for bringing this up. 

Since it sounds like RDBMSs normally don't take long to add a NULL column, how about we change this sentence as follows?:

```suggestion
- **For relational databases (MySQL, Oracle, etc.):** Adding a column shouldn't take a long time to execute. 
```

Or, do we still need to mention `In addition, a table lock may occur`?

Reply:
@josh-wong The same sentence is in the following:

https://github.com/scalar-labs/scalardb/blob/docs/revise-api-guide/docs/storage-abstraction.md#add-a-new-column-to-a-table
https://github.com/scalar-labs/scalardb/blob/docs/revise-api-guide/core/src/main/java/com/scalar/db/api/Admin.java#L392-L393

Can you please apply the same change to them?

Reply:
Sounds good. I'm currently editing `storage-abstraction.md`, so this change will be included when I submit the PR for that doc.

I've made the same changes in `Admin.java` in this PR, so PTAL!